src/webprojects/fxa.md

```diff
@@ -0,0 +1,124 @@
+
+* `LC_MESSAGES/client.po`: the content server for UI and frontend.
+* `LC_MESSAGES/server.po`: strings for emails and backend templates.
+* `Main.ftl`: payments for both frontend and backend.
+
+The above could change over time as the team is planning to migrate .po files to .ftl, which may involve restructuring the files.
+
+## Key links
+
+* [Production](https://accounts.firefox.com)
+* [Staging](https://latest.dev.lcip.org/)
+* [L10n Repository](https://github.com/mozilla/fxa-content-server-l10n)
+* [Pontoon](https://pontoon.mozilla.org/projects/firefox-accounts/)
+
+## Localizing the strings
+
+FxA lives in two file formats: .po and .ftl, both contain variable or placeable in the strings. The shorter the string, the more context you need to make sure you get the translation right. Look for clues or get clarifications through the following methods:
+
+* String comments: Pay attention to the comment or string ID. In Pontoon, the comments are displayed above the area where you can enter your translation for a string.
+* [Matrix channel](https://chat.mozilla.org/#/room/#l10n-community:mozilla.org): ping the L10n PM in charge with any issues you may have.
+* [Web projects mailing list](https://groups.google.com/g/mozilla.dev.l10n.web): Raise questions here so others may benefit from the information.
+
+## Errors in translations can break the build for all languages
+
+During translation, do not alter code in a string, such as variables and HTML markup.
+
+For .po file format, a few FxA contacts would be notified when code in a localized string is broken. Any subsequent commits by other localizers in other languages will trigger this warning until the broken string is fixed by one of the people notified. Here are a few common errors seen in localized content:
+
+**Example:** Alter the code by omitting or inserting space breaks the build.
+* Source: `The recipient’s consent is required. <a %(escapedLearnMoreAttributes)s>Learn more</a>`
+* Target:
+   * ❌ `A címzett hozzájárulása szükséges. <a%(escapedLearnMoreAttributes)s >Tudjon meg többet</a>.`
+   * ✅ `A címzett hozzájárulása szükséges. <a% (escapedLearnMoreAttributes)s>Tudjon meg többet</a>.`
+   * What changed in the broken translation (❌): space is removed from `<a% (` to `<a%(`; extra space was added from `)s>` to`)s >`.
+
+**Example:** Attributes should remain unchanged.
+* Source: `<span class="number">2</span>Point the camera at the QR code and tap the link that appears`
+* Target:
+   * ❌ `<span classe="numero">2</span>Puncta le camera al codice QR e tocca le ligamine que appare`
+   * ✅ `<span class="number">2</span>Puncta le camera al codice QR e tocca le ligamine que appare`
+   * What changed in the broken translation (❌): both the attribute's name `(class)` and value `(number)` were localized.
```

Comment:
Both couples of parenthesis should be outside of the code fence

Reply:
Sorry, completely missed this one. now it is fixed.